manifests/profile/tsm.pp

```diff
@@ -39,7 +42,8 @@
   Array[String] $inclexcl = ['exclude /.../.nfs*', 'exclude.dir /.../.snapshot'],
   Array[String] $domains = ['/etc','/opt','/var'],
   Array[String] $virtualmountpoints = ['/etc','/opt','/var'],
-  Array[String] $exclude_dirs = ['/afs/','/net/','/nfs/','/usr/vice/cache/']
+  Array[String] $exclude_dirs = ['/afs/','/net/','/nfs/','/usr/vice/cache/'],
+  Array[String] $opt_settings = ['* No custom settings']
```

Comment:
Given the default comment, would `$custom_settings` be a better name? `opt` feels needlessly abbreviated to me, while `optional` collides a bit with puppet's `Optional` syntax, but you say `* No custom settings` right there in the comment, so why not call it that?

Reply:
I thought about that. But there are two types of custom settings. Ones that go into "dsm.opt" and ones that go into "dsm.sys". So my intent was respective opt_settings and sys_settings (latter isn't there cause we don't have any custom setting for that yet).